README.md

```diff
@@ -92,6 +92,22 @@ Other operating systems, such as Microsoft Windows, which access files in a case
 
 Best practice is to use `data-e2e="your_test_id"` element attributes to select specific elements, eg `cy.get('[data-e2e="cta-button-1"]').click()` instead of `cy.get('.container .infobox a.button').contains('DOWNLOAD').click()`.
 
+|                       | Included in Cypress Test Production?     |  What is it used for?                                                                |
+|-----------------------|-|-------------------------------------------------------------------------------------------------------------------------------|
+| check_links.js        |✘ (takes a long time to finish running)   | Detect broken links throughout the website. Result is saved in `logs` folder         |
+| app_to_web.js         |✓                                         | Detect broken links throughout the website coming from cwa-webserver                |
```

Comment:
"Detect broken links throughout the website coming from cwa-webserver"

Change to:

"Check that all links to the website used by the Android and iOS apps are working and that content is correctly displayed."

Reply:
Changed!